apps/studio/src/components/connection/SqlServerForm.vue

```diff
@@ -177,6 +195,14 @@
             this.config.azureAuthOptions.azureAuthType = this.authType
           }
         }
+
+        const authId = this.config.azureAuthOptions?.authId || this.config?.authId
+        if (this.authType === AzureAuthType.AccessToken && !_.isNil(authId)) {
+          const cache = await TokenCache.findOne(authId);
```

Comment:
This will have to be moved to the utility process once we merge this to master

Reply:
Will do!